model/patch_lifecycle.go

```diff
@@ -649,7 +649,7 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 
 // getFullPatchParams will retrieve a merged list of parameters defined on the patch alias (if any)
 // with the parameters that were explicitly user-specified, with the latter taking precedence.
-func getFullPatchParams(p *patch.Patch) ([]patch.Parameter, error) {
+func getFullPatchParams(p *patch.Patch, config *ProjectConfig) ([]patch.Parameter, error) {
```

Comment:
config seems to be unused?

Reply:
ah thank you; i was considering a refactor but decided against it